platform/discord/compose.yaml

```diff
@@ -2,7 +2,8 @@ services:
   discord:
     build:
       context: .
-      target: final
+      tags:
+        - "pocketsizefund/discord:latest"
```

Comment:
I think this should be `discordbot`, right?

Reply:
yea, made the name consistent across the board